src/lib/config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
       GA4_ENABLED: process.env.GA4_ENABLED || "false",
       GA4_CONTAINER_ID: process.env.GA4_CONTAINER_ID,
     },
+    LANGUAGE_TOGGLE_ENABLED: process.env.LANGUAGE_TOGGLE_ENABLED || false,
```

Comment:
Is false the correct default value here if we're using `0` and `1` to enable?

Reply:
yeah I think you're right, it should be 0

Reply:
updated